src/diff/props.js

```diff
@@ -11,12 +11,17 @@ import options from '../options';
  */
 export function diffProps(dom, newProps, oldProps, isSvg) {
```

Comment:
Can `newProps` or `oldProps` be null or undefined here? If so it would help to clarify that in the function signature.

Reply:
Afaik they are never `null` here. When I remove the `!oldProps` check the byte-size goes up by `+7B` 🤷‍♂️

Reply:
Any chance this would trip for a vnode that swaps from Text to Element?

```
render(<div>foo</div>, root)
render(<div><span a /></div>, root)
```

Reply:
Good call, we need to keep that check :+1: